components/sensor/ruuvitag.rst

```diff
@@ -0,0 +1,177 @@
+
+.. figure:: images/ruuvitag-ui.jpg
+    :align: center
+    :width: 80.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    esp32_ble_tracker:
+
+    sensor:
+    - platform: ruuvitag
+      mac_address: FF:56:D3:2F:7D:E8
+      humidity:
+        name: "RuuviTag Humidity"
+      temperature:
+        name: "RuuviTag Temperature"
+      pressure:
+        name: "RuuviTag Pressure"
+      acceleration_x:
+        name: "RuuviTag Acceleration X"
+      acceleration_y:
+        name: "RuuviTag Acceleration Y"
+      acceleration_z:
+        name: "RuuviTag Acceleration Z"
+      battery_voltage:
+        name: "RuuviTag Battery Voltage"
+      tx_power:
+        name: "RuuviTag TX Power"
+      movement_counter:
+        name: "RuuviTag Movement Counter"
+      measurement_sequence_number:
+        name: "RuuviTag Measurement Sequence Number"
+
+Configuration variables:
+------------------------
+
+- **mac_address** (**Required**, MAC Address): The MAC address of the RuuviTag
+  device.
+- **humidity** (*Optional*): The information for the humidity sensor
+
+  - **name** (**Required**, string): The name for the humidity sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **temperature** (*Optional*): The information for the temperature sensor.
+
+  - **name** (**Required**, string): The name for the temperature sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **pressure** (*Optional*): The information for the pressure sensor.
+
+  - **name** (**Required**, string): The name for the pressure sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **acceleration_x** (*Optional*): The information for the acceleration x
+  sensor.
+
+  - **name** (**Required**, string): The name for the acceleration x sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **acceleration_y** (*Optional*): The information for the acceleration y
+  sensor.
+
+  - **name** (**Required**, string): The name for the acceleration y sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **acceleration_z** (*Optional*): The information for the acceleration z
+  sensor.
+
+  - **name** (**Required**, string): The name for the acceleration z sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **battery_voltage** (*Optional*): The information for the battery voltage
+  sensor
```

Comment:
```suggestion
  sensor.
```

This should probably also include an additional sentence like "Only supported for RAWv2" protocol (people don't tend to read the top part from my experience)

Reply:
I thought I mentioned it at the individual options...